monsieurbiz/sylius-sales-reports-plugin/1.0/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+    "bundles": {
+        "MonsieurBiz\\SyliusSalesReportsPlugin\\MonsieurBizSyliusSalesReportsPlugin": [
```

Comment:
`"MonsieurBiz\\SyliusSalesReportsPlugin\\MonsieurBizSyliusSalesReportsPlugin": ['all' => true]`

Reply:
Actually the recipe is correct. `=> true` is for the `bundles.php` in a symfony application, not in the manifest.